package.json

```diff
@@ -652,8 +681,15 @@
           "command": "zowe.uss.createNode",
           "group": "inline"
         },
+        {
+
+          "when": "viewItem == uss_session",
```

Comment:
Missed the use of regular expressions for viewitem "viewItem =~ /^(?!._fav.)ussSession.*/",

Reply:
oh men... I missed one. I am going to us regex "viewItem =~ /^ussSession.*/", because the command is under inline group. So I will paatern it with fullPath and createNode command